src/web/browser/ga/ga.ts

```diff
@@ -189,3 +189,26 @@ export const trackNonClickInteraction = (actionName: string): void => {
         );
     }
 };
+
+export const trackPerformance = (
+    timingCategory: string,
+    timingVar: any,
+    timingLabel: string,
+): void => {
+    const { ga } = window;
+
+    if (!ga) {
+        return;
+    }
+
+    if (window.performance?.now) {
+        ga(
+            send,
+            'timing',
+            timingCategory,
+            timingVar,
+            Math.round(window.performance.now()),
```

Comment:
Bold. Not even a `ceil`.

Reply:
it's what the GA docs use https://developers.google.com/analytics/devguides/collection/analyticsjs/user-timings#measuring_time